it/connectors/MessageConnectorSpec.scala

```diff
@@ -0,0 +1,98 @@
+package connectors
+
+import com.github.tomakehurst.wiremock.client.WireMock._
+import org.scalatest.concurrent.PatienceConfiguration.Timeout
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.matchers.should.Matchers
+import org.scalatestplus.play.guice.GuiceOneAppPerSuite
+import play.api.test.Helpers._
+import play.api.test.{FakeHeaders, FakeRequest, Helpers}
+import org.scalatest.matchers.must.Matchers.convertToAnyMustWrapper
+import org.scalatestplus.mockito.MockitoSugar
+
+import scala.concurrent.duration.Duration
+//import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.wordspec.AnyWordSpec
+import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse}
+
+class MessageConnectorSpec extends AnyWordSpec with Matchers with WiremockSuite with ScalaFutures with MockitoSugar {
```

Comment:
If you intend to keep the LOCKED status code, I think a test needs to be added here.